config/git-hooks/pre-push

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# Exit immediately in case of successful linting
```

Comment:
You mean unsuccessful.

Reply:
Fixed [here](https://github.com/Fezzzi/Household-Chores/pull/4/commits/987555e908bf6e1281a2ebaaadcd0758deeb4b6a#diff-90234faa0450b4e137d8b2f92fc69783)